### ISSUE TITLE
[Phase2] Mismatch reading/writing H

### ISSUE THREAD

**Issue by Brechtpd** [kobigurk/phase2-bn254#1]
There seems to be mismatch in the number of elements that are written to and then read from the `phase1radix2` files for H.
- When writing`degree - 1` elements are written [here](https://github.com/kobigurk/phase2-bn254/blob/350ccc6ee967d6f983dd6701bf36cc3bfa450410/powersoftau/src/bin/verify.rs#L346)
- When reading `degree` elements are read [here](https://github.com/kobigurk/phase2-bn254/blob/350ccc6ee967d6f983dd6701bf36cc3bfa450410/phase2/src/lib.rs#L515)

For me this results in the following error when reading the H elements running the examples:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: IoError(Custom { kind: UnexpectedEof, error: "failed to fill whole buffer" })'
```

Is there something I'm missing or is this indeed the problem?

**Comment by kobigurk** [comment]
Hey @Brechtpd, thanks for catching this.
It seems you're right, I probably missed one of the changes I should have committed.

Small point - we use https://github.com/kobigurk/phase2-bn254/blob/master/powersoftau/src/bin/prepare_phase2.rs rather than verify.rs, since it doesn't perform re-verification of all the previous contributions.

I'll test that and commit the right version in the coming days - if you want to test it before, feel free to get in touch.

**Comment by Brechtpd** [comment]
Fixed in #2.
